web-tester/pom.xml

```diff
@@ -0,0 +1,126 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.axonivy.ivy.webtest</groupId>
+  <artifactId>web-tester</artifactId>
+  <version>8.0.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  
+  <name>web-tester</name>
+  <description>API that simplifies web testing with ivy projects.</description>
+  <url>https://github.com/ivy-supplements/web-tester/</url>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>IvyTeam</name>
+      <email>support@ivyteam.ch</email>
+      <organization>AXON Ivy AG</organization>
+      <organizationUrl>https://www.axonivy.com</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git@github.com:ivy-supplements/web-tester.git</connection>
+    <developerConnection>scm:git:git@github.com:ivy-supplements/web-tester.git</developerConnection>
```

Comment:
will you rename the repo or what's the plan?

Reply:
I think this would make sense. Rename repo from primeui-tester to web-tester or something like that. Or do you have an different opinion?

Reply:
I support that change. I do not know whether github has some sort of 'redirect' or repo move features so that users of the old link will get a notification that the repo no longer resides in the old location. 
However, it's a third party project and I think rarely accessed via github. So even if the are is no move support we can live with that

Reply:
Seems that github redirects automatically to the new repo: https://help.github.com/en/github/administering-a-repository/renaming-a-repository